api/src/main/java/org/pmiops/workbench/config/WebMvcConfig.java

```diff
@@ -100,6 +104,7 @@ public void addCorsMappings(CorsRegistry registry) {
 
   @Override
   public void addInterceptors(InterceptorRegistry registry) {
+    registry.addInterceptor(corsInterceptor);
```

Comment:
Can we remove the CorsRegistry stuff now?

Reply:
I will check.